tests/suitespec.py

```diff
@@ -16,8 +16,8 @@ def get_patterns(suite: str) -> t.Set[str]:
     ['.circleci/*', 'ddtrace/__init__.py', 'ddtrace/_hooks.py', 'ddtrace/_logger.py', 'ddtrace/_monkey.py',
     'ddtrace/_tracing/*', 'ddtrace/auto.py', 'ddtrace/bootstrap/*', 'ddtrace/commands/*', 'ddtrace/constants.py',
     'ddtrace/context.py', 'ddtrace/debugging/*', 'ddtrace/filters.py', 'ddtrace/internal/*', 'ddtrace/pin.py',
-    'ddtrace/provider.py', 'ddtrace/py.typed', 'ddtrace/sampler.py', 'ddtrace/settings/__init__.py',
-    'ddtrace/settings/config.py', 'ddtrace/settings/dynamic_instrumentation.py',
+    'ddtrace/provider.py', 'ddtrace/py.typed', 'ddtrace/sampler.py', 'ddtrace/sampling_rule.py',
```

Comment:
All this movement is just me adding ddtrace/sampling_rule.py in there and making the string fit